Yet Another Blog Engine.iml

```diff
@@ -8,7 +8,13 @@
             <sourceFolder url="file://$MODULE_DIR$/app" isTestSource="false" />
             <sourceFolder url="file://$MODULE_DIR$/test" isTestSource="true" />
         </content>
-                <content url="file:///Applications/play-1.5.3/modules/docviewer">
+                <content url="file:///Applications/play-1.5.3/modules/secure">
```

Comment:
Usually we do not check-in our .iml or .ipr files, those give individual settings for the IDE. So I will want things set-up for me and you will want things custom to you.